src/containers/taskList.jsx

```diff
@@ -117,7 +131,16 @@ const TaskList = ({ style = {}, autoSelect = false }) => {
   //
 
   const nameRenderer = (node) => {
-    const icon = node.data.isGroup ? 'folder' : tasksTypes[node.data.taskType]?.icon
+    
+    const isActive = node.data.active
+    const isGroup = node.data.isGroup
+
+    const generateIcon = () => {
+      if (!isActive) return 'archive'
+      if (isGroup) return 'folder'
+      return tasksTypes[node.data.taskType]?.icon
+    }
+
     let className = ''
```

Comment:
Also lets skip the pairing and classname assignment so that `inactive` tasks don't get that pairing colour.

Reply:
![image](https://github.com/ynput/ayon-frontend/assets/49156310/71757f78-46ef-4b83-b704-4f128d4f8fcd)

Reply:
Done.